service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceMethod.java

```diff
@@ -148,7 +148,7 @@ private synchronized MethodResult invokeMethod(URL url, String methodName, Long
         }
 
         long  responseTimeout, connectTimeout;
-        
+
         if (responseTimeoutInSeconds == null)
```

Comment:
I want to make sure is there any breaking behavior change post this PR ? Also can you confirm Jobs and methods E2E pass with your change?